tests/ui-toml/toml_import_rename/conf_import_rename.rs

```diff
@@ -0,0 +1,13 @@
+#![warn(clippy::import_rename)]
+
+extern crate regex;
+extern crate serde;
+extern crate syn;
+
+use regex::bytes as xegersetyb;
+use serde::{de::Deserializer as WrongRename, ser::Serializer as DarkTower};
+use syn::Path as SynPath;
```

Comment:
Is using external crates for testing this really necessary? I'd like to avoid that.